### DISCUSSION TITLE
Where will uploaded VOD files be saved? Can I customize the directory location in the configuration?

### DISCUSSION THREAD

**Discussion by murazi** [ant-media/Ant-Media-Server#6064] [Announcements]
I am AMS deployed via Docker. Can I customize the directory location uploaded by VOD in the installation configuration?

**Comment by burak-58** [comment]
Hi @murazi,
This seems a question that you can ask in github discussion. I am converting it to a discussion. If at the end of the discussion we need to make it an issue we can get back.

**Comment by yashtandon113** [comment]
Hi @murazi 
You can use the VOD upload user-defined script to move the uploaded file from one folder to another.

- In order to use the user defined script for VOD upload, checkout [this](https://antmedia.io/docs/guides/recording-live-streams/user-defined-scripts/#vod-upload-finish-process).

- So you can create the script in any folder and you need to pass the path of the script into the above-mentioned application settings.

You can create the bash script as follows:

```
#!/bin/bash

file="$1"
file_name=$(basename "$file" .mp4)

# Move the file to the destination folder

mv "$file" /usr/local/antmedia/recordings
```

This script will be triggered right after the VOD is uploaded.

- After this file is moved from the default streams directory, on the web panel, you will still see the VOD file record, but it will not play as the file has already been moved to a different folder in the backend. You can also delete that with the help of [VOD delete Rest API](https://antmedia.io/rest/#/VoD%20Rest%20Service/deleteVoD)

I hope it helps. Please feel free to ask if you have any questions.
